### DISCUSSION TITLE
MTEX Figure ColorKeys Being Chopped Off After Generation

### DISCUSSION THREAD

**Discussion by Savillian** [mtex-toolbox/mtex#1174] [Ask Anything]
Hi everyone,

Apologies if this is a relatively easy fix, but I've found no standard MATLAB solution to this problem and the MTEX forums seem empty of this issue as far I could find.

I'm trying to generate Schmid factor figures with a set color range. Whenever I generate one of these (see below), the color bar scale is partially chopped off at the ends and no resizing fixes the issue.

![A3Y10BetaSchmidSlip](https://user-images.githubusercontent.com/38051659/129082203-ca2c603f-15c5-4f5b-a6ad-bba9588f6c09.png)

In the past I've had success getting around similar issues by using `figure('Renderer', 'painters', 'Position', [10 10 1920 1080])`, but this only works for figures not created using MTEX commands. Because of how this plot is generated, this and similar solutions have proven ineffective.

The color bar in question is affected by the MTEXColorbar command, and I can readily change the position as need be. However, the color bar always is partially chopped off on the ends regardless of the options I've been able to apply. This  _only_ occurs if I specify the range for the color scheme however. Does anyone know of a workaround either via MTEX or using a MATLAB command that interfaces with these plots?

An example script is included below:
`% plot the maximum Schmid factor
figure('Renderer', 'painters', 'Position', [10 10 1920 1080])
plot(Reconstructedgrains,SFMaxb,'micronbar','off','linewidth',1,'NoFrame')
mtexColorbar location southoutside
colormap hot
CLim(gcm,[0.4,0.5]);
`
Thanks in advance,
Alec

**Comment by kilir** [comment]
Hi,
you can try to increase the outerPlotSpacing `setMTEXpref('outerPlotSpacing',60)`.
Cheers,Rüdiger
